subpackages/runner/source/unit_threaded/runner/testsuite.d

```diff
@@ -166,12 +166,9 @@ private:
 
         _stopWatch.start();
 
-        const numThreads = _options.numThreads;
-        if (numThreads != 1) {
+        if (_options.numThreads != 1) {
             // use a dedicated task pool with non-daemon worker threads
-            auto taskPool = numThreads == 0
-                ? new TaskPool()                // use all logical CPU cores
-                : new TaskPool(numThreads - 1); // main thread is used too
+            auto taskPool = new TaskPool(_options.numThreads);
```

Comment:
This is a functional change. I suggest the `-1` here and defaulting to `totalCPUs`.

Reply:
Yes, but that's the only thing that makes sense to me.

Reply:
It's now using N+1 threads when using `-j N`, as the main thread is used too AFAIU.